spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-sample/pom.xml

```diff
@@ -9,7 +9,13 @@
 	</parent>
 
 	<artifactId>spring-cloud-gcp-integration-pubsub-sample</artifactId>
-	<groupId>com.google.cloud</groupId>
```

Comment:
Wait, how did Pub/Sub get tangled into this? You will want to revert this file from the pull request. Easiest way is to do "git > compare with branch > main", and accept changes from main.